go/signedexchange/signedexchange.go

```diff
@@ -168,36 +168,44 @@ func (e *Exchange) encodeExchangeHeaders(enc *cbor.Encoder) error {
 
 // draft-yasskin-http-origin-signed-responses.html#application-http-exchange
 func WriteExchangeFile(w io.Writer, e *Exchange) error {
-	enc := cbor.NewEncoder(w)
-	if err := enc.EncodeArrayHeader(7); err != nil {
-		return err
-	}
-	if err := enc.EncodeTextString("htxg"); err != nil {
-		return err
-	}
-
-	if err := enc.EncodeTextString("request"); err != nil {
+	buf := &bytes.Buffer{}
+	enc := cbor.NewEncoder(buf)
+	if err := enc.EncodeArrayHeader(2); err != nil {
 		return err
 	}
 	// FIXME: This may diverge in future.
 	if err := e.encodeRequestWithHeaders(enc); err != nil {
 		return err
 	}
-
 	// FIXME: Support "request payload"
```

Comment:
This comment is obsolete. Remove the line

Reply:
Done.